exo-clone/src/app-cloner.ls

```diff
@@ -24,11 +24,11 @@ class AppCloner extends EventEmitter
       config-path = path.join @repository.path, 'application.yml'
       @app-config = yaml.safe-load fs.read-file-sync(config-path, 'utf8')
       @logger.set-colors Object.keys(@app-config.services)
-      cloners = for type of @app-config.services
-        for service-name, service-data of @app-config.services[type]
+      cloners = for protection-type of @app-config.services
```

Comment:
If "protection refers to "public" and "private", let's use "protection-level" here.